plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -228,6 +227,14 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
    cfg.add_options()
          ("blocks-dir", bpo::value<bfs::path>()->default_value("blocks"),
           "the location of the blocks directory (absolute path or relative to application data dir)")
+         ("blocks-log-stride", bpo::value<uint32_t>()->default_value(config::default_blocks_log_stride),
```

Comment:
I think we should add here (and also in PR API documentation) something like "When the stride is breached, the current blog log and index will be renamed 'blocks-<lowest block num>-<highest block num>.log/index' and a new current block log and index will be created with the most recent block. All files following this format will be used to construct an extended block log."